electrum/commands.py

```diff
@@ -1258,6 +1258,7 @@ def subparser_call(self, parser, namespace, values, option_string=None):
 
 
 def add_network_options(parser):
+    parser.add_argument("-f", "--fingerprint", dest="fingerprint", default=None, help="only allow connecting to servers with a matching SSL certificate SHA256 fingerprint")
```

Comment:
`fingerprint` sounds a bit too generic, it might become ambiguous in the future.
maybe `sslfingerprint` or `serverfingerprint` or `sslcerthash`? well not sure.

Reply:
I agree, I like `serverfingerprint`.

Reply:
Resolved by https://github.com/spesmilo/electrum/pull/6094/commits/0fcde1306197927de3793326aba840df2ad7c80a